src/main/java/cceclipseplugin/core/PluginManager.java

```diff
@@ -35,19 +62,18 @@
 	private static PluginManager instance;
 
 	// PLUGIN SETTINGS (will be moved to preferences later)
-	// final private String WS_ADDRESS = "ws://echo.websocket.org";
-	final private String WS_ADDRESS = "ws://localhost:8000/ws/";
+	final private String WS_ADDRESS = "ws://cody.csse.rose-hulman.edu:8000/ws/";
```

Comment:
Wouldn't this be a configuration thing, not a preferences entry? Preferences are generally for things the user can change. We could move it there for testing, but in my opinion, this is counteracting our 'productionization' efforts. I do agree we should put it in a configuration file of some kind that the plugin has access to. I've made a card for that; I don't think it is a change for this pull request.